ibmcloudant/cloudant_base_service.py

```diff
@@ -65,6 +70,9 @@ def __hash__(self):
 
 def new_init(self, authenticator: Authenticator = None):
     old_init(self, authenticator)
+    # Overwrite default read timeout to 2.5 minutes
+    if not ('timeout' in self.http_config):
+        self.set_http_config({'timeout':Timeout(connect=CONNECT_TIMEOUT, read=READ_TIMEOUT)})
```

Comment:
Doesn't this overwrite the config dict with a new one containing _only_ the timeout? The easy thing would be to set `http_config['timeout']` but I think it is clear that we need to call `set_http_config` as well to ensure the update is applied to the authenticators.

Secondly, whilst this is a `urllib3` timeout under the hood, we are actually dealing with the python sdk core config dict here, which in turn gets made into `kwargs` that eventually get passed to `requests` and finally on to `urllib3` - I'd prefer we just used a tuple here rather than the `Timeout` type as that is what [`requests` expects](https://docs.python-requests.org/en/latest/user/advanced/#timeouts) and that is the API that is exposed to us and it will also save us an import pain:

```suggestion
        new_http_config = self.http_config.copy()
        new_http_config['timeout] = (CONNECT_TIMEOUT, READ_TIMEOUT)
        self.set_http_config(new_http_config)
```

Reply:
Indeed, thanks for the suggestion: 24bad01